doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -61,7 +59,7 @@ The following list summarizes the most important changes inherited from upstream
 Mcumgr
 ======
 
-The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``e64f5060b``.
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``657deb65``.
```

Comment:
Same here, why is the SHA changing?

Reply:
This changed at the last minute before the release (we updated the SHA together with you and @SebastianBoe ), so I'm mirroring what is in the 1.7.0 release notes.